meeting-notes/20181113.md

```diff
@@ -0,0 +1,203 @@
+   - discussify:
+     - UI level QA: https://github.com/ipfs-shipyard/pm-discussify/issues/58
+     - Standardized styleguide comment names with @andreforsousa
+ - In progress:
+   - Replies
+   - Lazy load of comments and replies (view more)
+ - Next:
+   - Implement "show history" in the UI
+   - Add persistance of the data (pinner + sync UI language)
+     
+Note: Ask pedro about granularity of Discussify v1 milestones on the roadmap
+     
+ #### @andreforsousa    
+ - Concluded:
+    - Discussify:
+    - Style-guide updates with all scenarios and states (hover, focus, error feedback) for "comment-component"
+        - Comment
+        - Comment (ownership)
+        - Comment Actions
+        - Comment Edited
+        - Comment Edited History
+        - Comment Placeholder
+        - Comment Removed
+        - Comment Input (new reply)
+        - Comment Input (editing)
+    - QA & style-guide updates for the pre-alpha version
+        - FAB Shadow fix
+    - PeerPad UX/UI fixes:
+        - Grid width for 80columns accommodation (desktop)
+        - Dismissible notifications (desktop)
+        - Empty state illustration
+        - History revision
+        
+- Blocked:
+    - PeerPad mobile:
+    - Hi-Fi prototype to start looking for:
+        - Transitions between elements
+        - Start defining values for the eases
+        - This will be in the backlog until desktop version is properly defined and polished, following David’s feedback.
+    
+- In progress:
+  - Discussify Brand & Web Guidelines (manual)
+  - Look at new features for PeerPad (sync with Victor)
+
+ 
+#### @VictorBjelkholm
+ - Concluded:
+ 		- Error reporting on peerpad.net
+ - In Progress:
+ 		- UX review
+    - Pinner review
+ - Blocked:
+ - Next:
+    - Pinner deployment
+    - Metrics
+ 
+#### @aschmahmann
+ - Concluded:
+  - Roadmapping round #1
+  - Basic graph sync + updating working
+ - In progress:
+  - Looking into using existing CRDT tools like y.js to talk to the local Go graph syncing client
+     - Any advice would be appreciated
+ - Blocked:
+  - Not sure how much work to put into improving graph syncing vs wait for IPLD selectors/GraphSync
+  - A little blocked by the lack of the Wallet and Friends components of the Sharing API
+ - Next:
+  - More progress towards a causal chat application
+   - Which avenue to explore next somewhat depends on our file sharing API priorities
+    - Could also focus on writing proposals for this
+
+#### @jimpick
+ - Concluded:
+   - working group 2019 roadmap, 1st iteration (participant)
+   - fixes to ORMap CRDT
+   - Pedro's editor binding fixes
+   - real-life testing of peer-pad-nano with PL staff
+   - zombie gossip messages and hacks to filter them out
+   - React.js binding fixes (didn't work on Android!)
+   - Trying out the peer-pad e2e load tests
+   - Wildcard DNS / more PL staff testing
+ - In progress:
+   - Moving changes to production version (too diverged right now!)
```

Comment:
Which changes?

Reply:
There are a lot of little diffs, most of which are non-keepers or shortcuts, or extra logging/tracing code to assist my understanding. In some cases the changes are just wrong because I didn't understand the code yet (especially around pinning, which has changed).

I do want to change the store implementation ... the current store implementation using a leveldb abstraction on indexeddb in the browser is pathological. In peer-pad-nano, it's using a simple in-memory store, but that causes other problems because there's no persistence client-side.

https://github.com/ipfs-shipyard/peer-star-app/issues/117

I'm just getting into the automated testing work that @pgte started. I'm wondering if it might be possible to automatically capture profile data as part of the test? It would be nice to have a failing test for test cases like "a single keystroke on a remote peer should be handled with less than 10 event loops".